jb/fftw/time_delay_estimator_many.hpp

```diff
@@ -128,18 +133,17 @@ class time_delay_estimator_many {
 
     // ... finally we compute the estimated delay and its confidence
     // @todo issue #86: investigate use of SSE instructions
-    std::size_t k = 0;
     precision_type* it_out = out_.data();
     for (std::size_t i = 0; i != num_timeseries_; ++i) {
       precision_type max_val = std::numeric_limits<precision_type>::min();
       std::size_t tde_val = 0;
-      for (std::size_t j = 0; j != nsamples_; ++j, ++k, ++it_out) {
+      for (std::size_t j = 0; j != nsamples_; ++j, ++it_out) {
         if (max_val < *it_out) {
           max_val = *it_out;
           tde_val = j;
         }
       }
-      if (jb::testing::check_close_enough(sum2[i], precision_type{0}, 1)) {
+      if (sum2[i] < std::numeric_limits<precision_type>::epsilon()) {
         confidence[i] = std::numeric_limits<precision_type>::max();
```

Comment:
I did not catch this earlier, but do we really want confidence to be max when basically we have 0 information?  I would think confidence == 0 would be more appropriate.